### PR TITLE
Feature/audio fixes

### PR DESCRIPTION
**Pull request by RickyVimon**
What's new?
- SequenceController child on the AudioMannager to reset de sequences playing when respawning.
- Lvl2 ambient and crustal ambient sound (Corrupted/purified)

*Delete library before compiling

### CODE REVIEW

**Review by Dartemiss — APPROVED** [review]
AMAZING SPIDERMAN!

File: Scripts/src/Script/AudioManager.h [hunk 1]
```diff
@@ -24,10 +24,13 @@ class AudioManager : public Script
 private:
 	void PlayAudioOnAwake();
 
+	void GetSequenceControler();
```
Comment (Pacasasgar) [inline Scripts/src/Script/AudioManager.h:27]: This function is not defined in the '.cpp'. Delete it if not being used.

Comment (RickyVimon) [reply Scripts/src/Script/AudioManager.h]: solved

**Comment by Pacasasgar** [comment]
Appart from the comments I told you (solved already), very good job! Approving.

**Review by Pacasasgar — APPROVED** [review]

**Pull request merged**